models/issues/pull.go

```diff
@@ -379,6 +380,7 @@ func (pr *PullRequest) getReviewedByLines(ctx context.Context, writer io.Writer)
 		Types:        []ReviewType{ReviewTypeApprove},
 		IssueID:      pr.IssueID,
 		OfficialOnly: setting.Repository.PullRequest.DefaultMergeMessageOfficialApproversOnly,
+		Dismissed:    optional.Some(false),
```

Comment:
Why the logic changed and not displayed dismissed code reviews?

Reply:
-> https://github.com/go-gitea/gitea/pull/35500